lib/hawkular/inventory/entities_v4.rb

```diff
@@ -65,20 +65,17 @@ class Resource
     attr_reader :properties
     # @return [List<Metric>] Metrics associated to this resource
     attr_reader :metrics
-    # @return [List<String>] List of children ids (might be null if 'self.children' exists)
-    attr_reader :children_ids
-    # @return [List<Resource>] List of children (might be null if 'self.children_ids' exists)
-    attr_reader :children
+    # @return [String] Id of the entity's parent
+    attr_reader :parent_id
```

Comment:
Actually, there's no "parent_id" here because the model used to insert data is not exactly the same as the one returned on queries.